trello/search.md

```diff
@@ -0,0 +1,82 @@
+# Searching in Trello
+
+## Keyboard Shortcuts
+
+`/` puts the cursor in the search box
+`b` opens the boards menu
+`f` opens the card filter menu to search through cards on a single board
+`q` toggles "cards assigned to me"
+`x` clears all active card filters
+
+## Finding an archived card
+
+Menu -> More -> Archived Items
+- opens a search box restricted to archived cards
+
+## Search Operators
+
+Combine these in any search/filter menu (including archive) for a power-search experience
+
+-operator
+- e.g. -has:members to search for cards without any members assigned
+
+@name
+- Returns cards assigned to a member. member: also works.
+- @me will include only your cards.
+
+label:
+- Returns labeled cards. #label also works.
+
+board:id
+- Returns cards within a specific board.
+
+board:name
+- e.g “board:trello” returns cards on boards with trello in the board name.
+
+list:name
+- Returns cards within the list named “name”. Or whatever you type besides “name”.
+
+has:attachments
+- Returns cards with attachments.
+
+has:description, has:cover, has:members, and has:stickers also work as you would expect.
+
+due:day
+- Returns cards due within 24 hours.
+due:week
+- returns cards that are due within the following 7 days.
+- due:month, and due:overdue also work as expected.
+due:14
+- will include cards due in the next 14 days.
+due:complete or due:incomplete
```

Comment:
These "due" lines are rendering as one long line rather than individual lists:

![image](https://user-images.githubusercontent.com/18957/63268783-e1bd4b80-c28c-11e9-8634-c7e13cb77fae.png)

An empty line before each "due:..." will fix it :)